packages/cli/src/ErrorHandling.ts

```diff
@@ -0,0 +1,47 @@
+import * as Sentry from '@sentry/node';
+import { RewriteFrames } from '@sentry/integrations';
+import type { Application } from 'express';
+import config from '../config';
+import { ErrorHandler } from 'n8n-workflow';
+
+let initialized = false;
+
+export const initErrorHandling = (app?: Application) => {
+	if (initialized) return;
+
+	if (!config.getEnv('diagnostics.enabled')) {
+		initialized = true;
+		return;
+	}
+
+	const dsn = config.getEnv('diagnostics.config.sentry.dsn');
+	const { N8N_VERSION: release, ENVIRONMENT: environment } = process.env;
+
+	Sentry.init({
```

Comment:
Should we log out failed init?

Reply:
`Sentry.init` should actually never fail. But I've added logging for when error reporting is called before `init` has been called. In that case the actual error is also simply written to the console.